src/profile_data_collector.h

```diff
@@ -37,47 +37,47 @@
 
 namespace triton { namespace perfanalyzer {
 
-/// Data structure to hold which inference load mode was used for an experiment.
-/// Only one data member will be nonzero, indicating the inference load mode for
-/// a particular experiment.
-struct InferenceLoadMode {
```

Comment:
As mentioned, the `InferenceLoadMode` struct that is used for `ProfileDataCollector` got moved from the `triton::perf_analyzer` namespace into the `ProfileDataCollector` class

So that `triton::perf_analyzer::InferenceLoadMode` could be used for the enum holding what mode the user wants from CLI option parsing (e.g. concurrency, request rate, fixed schedule, etc).